.github/workflows/docker_all-modules.yml

```
@@ -69,7 +69,7 @@ jobs:
         if: contains(needs.*.result, 'failure')
         uses: peter-evans/create-issue-from-file@v5
         with:
-          title: Analysis module failing in CI
+          title: Docker image failing in CI
```

Might as well go one step further:
```suggestion
          title: Docker image build failing in CI
```